pydantic/_internal/_generate_schema.py

```diff
@@ -1476,11 +1476,13 @@ def _common_field(
     }
 
 
-def generate_computed_field(d: dict[str, Decorator[ComputedFieldInfo]]) -> list[core_schema.ComputedField] | None:
+def generate_computed_field(
```

Comment:
Does it make sense to just make this a method of `GenerateSchema`? That's what I did in https://github.com/pydantic/pydantic/pull/5708 but I'm happy to keep your approach of still returning a list. Not a big deal.

Reply:
done